src/compat/eme/custom_media_keys/webkit_media_keys_constructor.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2015 CANAL+ Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+type IWebKitMediaKeys = unknown;
+
+interface IWebKitMediaKeysConstructor {
+  new(keySystem: string) : IWebKitMediaKeys;
+  isTypeSupported: (keyType: string) => boolean;
+  prototype: { createSession: () => void };
```

Comment:
That's ugly as f***.
Doesn't it work if we go through `IWebkitMediaKeys` to set its method?

Also WebkitMediaKeys's createSession returns `undefined`?

Reply:
I don't understand why no-unsafe-any would be needed here :|